package.json

```diff
@@ -14,13 +14,14 @@
     "angular-mocks": "^1.3.15",
     "babel-core": "^5.4.2",
     "babel-loader": "^5.0.0",
-    "browser-sync": "^2.7.1",
```

Comment:
Fixed. Added `webpack-dev-middleware` and `webpack-hot-middleware` on top of browsersync.